ts/util/reactions.ts

```diff
@@ -86,7 +86,7 @@ const sendMessageReaction = async (messageId: string, emoji: string) => {
       return undefined;
     }
 
-    let me = UserUtils.getOurPubKeyStrFromCache();
+    let me: string = UserUtils.getOurPubKeyStrFromCache();
```

Comment:
Not PubkeyType?

Reply:
we can't, just below we do 
me = conversationModel.getUsInThatConversation();
which returns a string currently (should be BlindedPk or PubKeyType, not now is not the time)